FightPandemics/Features/TabBar/RootTabBarController.swift

```diff
@@ -143,4 +181,33 @@ extension RootTabBarController: UITabBarControllerDelegate {
         return true
     }
 
+    override func tabBar(_ tabBar: UITabBar, didSelect item: UITabBarItem) {
+        item.setTitleTextAttributes([NSAttributedString.Key.foregroundColor: UIColor.clear], for: .selected)
+        tabBar.tintColor = .black
+        let itemView = item.value(forKey: "view") as? UIView
+        var addend = CGFloat()
+        if tabBar.items?.firstIndex(of: item) == 0 {
+            addend = -19
+        } else if tabBar.items?.firstIndex(of: item) == 1 {
+            addend = -24
+        } else if tabBar.items?.firstIndex(of: item) == 2 {
+            addend = 18
+        } else if tabBar.items?.firstIndex(of: item) == 3 {
+            addend = 12
+        }
+        let dotView = UIView()
+        if tabBar.viewWithTag(100) != nil {
+            tabBar.viewWithTag(100)?.removeFromSuperview()
+        }
+        dotView.backgroundColor = .blue
```

Comment:
doesnt seem the same as figma

figma is 425AF2

https://jmp.sh/nRPYj5k

Reply:
right!!